### ISSUE TITLE
Fix accessibility test errors for <Controls> component 

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/mima#112]
It returns the following errors. For the time being, the test is skipped.
```
Expected the HTML found at $('button[data-position="top-right"]') to have no violations:

    <button data-position="top-right" type="button" class="Button-sc-4v1wtd-0 cpKrVf">

    Received:

    "Buttons must have discernible text (button-name)"

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element's default semantics were not overridden with role="none" or role="presentation"
      Element has no title attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/button-name?application=axeAPI

    Expected the HTML found at $('button[data-position="bottom-right"]') to have no violations:

    <button data-position="bottom-right" type="button" class="Button-sc-4v1wtd-0 cpKrVf">

    Received:

    "Buttons must have discernible text (button-name)"

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element's default semantics were not overridden with role="none" or role="presentation"
      Element has no title attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/button-name?application=axeAPI

    ────────

    Expected the HTML found at $('svg[aria-labelledby="search"][height="\33 6px"][viewBox="\30 \ 0\ 24\ 24"] > title') to have no violations:

    <title id="search"></title>

    Received:

    "IDs used in ARIA and labels must be unique (duplicate-id-aria)"

    Fix any of the following:
      Document has multiple elements referenced with ARIA with the same id attribute: search

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/duplicate-id-aria?application=axeAPI

    Expected the HTML found at $('svg[aria-labelledby="add"][height="\33 6px"][viewBox="\30 \ 0\ 24\ 24"] > title') to have no violations:

    <title id="add"></title>

    Received:

    "IDs used in ARIA and labels must be unique (duplicate-id-aria)"

    Fix any of the following:
      Document has multiple elements referenced with ARIA with the same id attribute: add

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/duplicate-id-aria?application=axeAPI

    ────────

    Expected the HTML found at $('svg[aria-labelledby="menu"] > path:nth-child(2)') to have no violations:

    <path d="M45.4620546,13.6147645 C51.6790144,16.4506152 56,22.7206975 56,30 C56,39.9411255 47.9411255,48 38,48 C32.9385058,48 28.3649488,45.9108926 25.09456,42.5479089 C22.9175971,43.482463 20.5192372,44 18,44 C8.0588745,44 0,35.9411255 0,26 C0,17.9805361 5.24437759,11.1859622 12.4906291,8.85878199 C15.6225135,3.55654277 21.3959192,0 28,0 C36.428553,0 43.5040602,5.79307725 45.4620546,13.6147645 Z" id="cloud">

    Received:

    "id attribute value must be unique (duplicate-id)"

    Fix any of the following:
      Document has multiple static elements with the same id attribute: cloud

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/duplicate-id?application=axeAPI

    ────────

    Expected the HTML found at $('svg[aria-labelledby="search"][width="\35 6px"][height="\34 8px"]') to have no violations:

    <svg role="img" aria-labelledby="search" width="56px" height="48px" viewBox="0 0 56 48">

    Received:

    "svg elements with an img role have an alternative text (svg-img-alt)"

    Fix any of the following:
      Element child title is empty
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/svg-img-alt?application=axeAPI

    Expected the HTML found at $('svg[aria-labelledby="add"][width="\35 6px"][height="\34 8px"]') to have no violations:

    <svg role="img" aria-labelledby="add" width="56px" height="48px" viewBox="0 0 56 48">

    Received:

    "svg elements with an img role have an alternative text (svg-img-alt)"

    Fix any of the following:
      Element child title is empty
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.1/svg-img-alt?application=axeAPI
```
